node_client_example/src/client.js

```diff
@@ -35,16 +35,26 @@ var ajaxSettings = {
     // Basic auth params are OK (hardcoding these for the demo nginx proxy)
```

Comment:
I think removing this comment should be removed now since it's not really applicable anymore now that the username/pass are configurable and configured below. Can you update to initialize an empty ajaxSettings object and add comments below about the optional user/pass for basic auth?